### PR TITLE
Feature/coming soon module

### PR DESCRIPTION
**Pull request by circlecube**
Remove coming soon code.
Add coming soon as a module.
Configure coming soon module with strings and plugin-specific stylesheet.

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -19,6 +22,66 @@
...
+			);
+		}
+	)
+);
+
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
```
Comment (wpscholar) [inline bootstrap.php:50]: Can we just add an upgrade routine to use `nfd_coming_soon` instead?

File: bootstrap.php [hunk 2]
```diff
@@ -19,6 +22,66 @@
...
+		'admin_screen_id'     => 'hostgator',
+		'admin_app_url'       => admin_url( 'admin.php?page=hostgator#/home' ),
+		'template_h1'         => 'A New WordPress Site',
+		'template_h2'         => 'Coming Soon!',
```
Comment (wpscholar) [inline bootstrap.php:54]: The two strings above need to be translatable.

File: bootstrap.php [hunk 3]
```diff
@@ -19,6 +22,66 @@
...
+		}
+	)
+);
+
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
+		'admin_screen_id'     => 'hostgator',
```
Comment (wpscholar) [inline bootstrap.php:51]: Can we just use `container()->plugin()->id` as the default in the coming soon module?

File: bootstrap.php [hunk 4]
```diff
@@ -19,6 +22,66 @@
...
+			__( '%s &mdash; Coming Soon', 'wp-plugin-hostgator' ),
+			esc_html( get_option( 'blogname' ) )
+		),
+		'admin_bar_text'      => '<div style="background-color: #ffcf00; color: #191936; padding: 0 1rem;">' . __( 'Coming Soon Active', 'wp-plugin-hostgator' ) . '</div>',
+		'admin_notice_text'   => sprintf(
+			/* translators: %1$s is replaced with the opening link tag and %2$s is replaced with the closing link tag, %3$s is the opening link tag to preview the page, %4$s is the closing link tag. */
+			__( 'Your site is currently displaying a %3$sComing Soon page%4$s. Once you are ready, %1$slaunch your site%2$s.', 'wp-plugin-hostgator' ),
+			'<a href="' . esc_url( admin_url( 'admin.php?page=hostgator#/home' ) ) . '">',
+			'</a>',
+			'<a href="' . get_home_url() . '?preview=coming_soon" title="Preview the Coming Soon landing page">',
```
Comment (wpscholar) [inline bootstrap.php:76]: Title attribute should be translatable here

**Review by wpscholar — CHANGES_REQUESTED** [review]

File: bootstrap.php (hunk 1 above)
Comment (circlecube) [reply bootstrap.php]: Done

File: bootstrap.php (hunk 2 above)
Comment (circlecube) [reply bootstrap.php]: Yes, added the `__`

File: bootstrap.php (hunk 3 above)
Comment (circlecube) [reply bootstrap.php]: Yes, this is in place now.

File: bootstrap.php (hunk 4 above)
Comment (circlecube) [reply bootstrap.php]: Got it!

**Pull request merged**